app/tests/reader_studies_tests/test_views.py

```diff
@@ -931,3 +931,47 @@ def test_display_set_upload_corrupt_image(
     notification = Notification.objects.get()
     assert notification.user == editor
     assert "1 file could not be imported" in notification.description
+
+
+@pytest.mark.django_db
+def test_ground_truth_view(client):
+    rs = ReaderStudyFactory()
+
+    editor, reader, a_user = UserFactory.create_batch(3)
+    rs.add_editor(editor)
+    rs.add_reader(reader)
+
+    for usr in [reader, a_user]:
+        response = get_view_for_user(
+            client=client,
+            viewname="reader-studies:ground-truth",
+            reverse_kwargs={"slug": rs.slug},
+            user=usr,
+        )
+        assert response.status_code == 403
+
+    response = get_view_for_user(
+        client=client,
+        viewname="reader-studies:ground-truth",
+        reverse_kwargs={"slug": rs.slug},
+        user=editor,
+    )
+
+    educational_warning = (
+        "This reader study is currently not configured as educational"
```

Comment:
We could hide the link if it is not educational?

Reply:
Mhhh. That would also hide the feature. Given that most people don't have the best track record of reading documents or feature lists I am generally fond of showing a disabled action unless the UI clutter calls for it.

Then again, this is really only applicable to ppl who want to have an educational reader study. Which is a minority of the reader studies and the feature **is** documented here: https://grand-challenge.org/documentation/creating-questions/#educational-reader-studies .

We could then also hide the leaderboard link.

Locking away the deletion of ground truth when a reader study is not not educational seems an OK side effect. I think?

Reply:
Done! Both are now hidden.